docroot/modules/custom/mass_fields/mass_fields.module

```diff
@@ -726,6 +726,9 @@ function mass_fields_entity_field_access($operation, FieldDefinitionInterface $f
       case 'field_details_data_type':
       case 'field_binder_data_type':
       case 'field_list_data_type':
+        // See DP-26260: Add field to data metadata fields so we can
+        // paste in additional content to search.
+      case 'field_data_hub_search_content':
```

Comment:
@sagar-lcm looks like this just needs to be updated
```suggestion
      case 'field_data_search_content':
```

Reply:
@tom-fleming Thank you for pointing this out, I forgot about it.